spec/spec_helper.rb

```diff
@@ -3,7 +3,7 @@
 require "logstash/inputs/ganglia"
 require_relative "support/client"
 
-module GangliaHelpers
+class GangliaHelpers
 
   def setup_clients(number_of_clients, port)
     number_of_clients.times.inject([]) do |clients|
```

Comment:
`setup_clients` is never used here, so this means we can remove the Helper class